theme/static/css/hydroshare_core.css

```diff
@@ -4345,4 +4345,10 @@ div#fb-file-operations-controls {
 
 #rename-dialog .input-group {
 	width: 100%;
+}
+
```

Comment:
If the icon at theme/static/img/loader.gif is no longer used we might want to delete it.